cloudevents/bindings/nats-protocol-binding.md

```diff
@@ -114,6 +114,8 @@ If the content mode is _structured_ then the NATS message payload MUST be
 the [JSON event format][json-format] serialized as specified by the
 [UTF-8][rfc3629] encoded JSON text for use in NATS.
 
+The NATS protocol binding does not currently support the _batch_ content mode.
```

Comment:
Any reason you didn't use the same text we saw in the above binding specs? E.g. the stuff about "the spec defines 3....". I like consistency :-)

Reply:
Definitely could do - I was trying to be somewhat minimal. Will wait until the meeting before making changes.

Reply:
It was in entirely the wrong please, too!

Reply:
Done in the next commit.